client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -23,23 +23,29 @@ import type {
 const INITIAL_UNIX_EPOCH = '1970-01-01 00:00:00';
 
 const isExpandedBlockEnabled = config.isEnabled( 'jetpack/pro-dashboard-expandable-block' );
+const isBoostEnabled = config.isEnabled( 'jetpack/pro-dashboard-jetpack-boost' );
 
 // Mapping the columns to the site data keys
 export const siteColumnKeyMap: { [ key: string ]: string } = {
 	site: 'url',
 };
 
-const extraColumns: SiteColumns = isExpandedBlockEnabled
+const boostColumn: SiteColumns = isBoostEnabled
 	? [
 			{
-				key: 'stats',
-				title: translate( 'Stats' ),
+				key: 'boost',
+				title: translate( 'Boost' ),
 				className: 'width-fit-content',
 				isExpandable: true,
 			},
+	  ]
+	: [];
+
+const extraColumns: SiteColumns = isExpandedBlockEnabled
```

Comment:
I think we can rename `extraColumns` as `StatsColumn` now. WDYT?

Reply:
This makes sense to me and I've made the change